assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -24,11 +24,11 @@
 }
 
 .nav-primary__item {
-  float: left;
   @include nav-typeg();
-  list-style-type: none;
   @include padding(9 12 0 0);
-  text-transform: uppercase;
+  color: #292929;
```

Comment:
Can we double check with Chris that this is a new colour being introduced as it's not been used before